manifests/database.pp

```diff
@@ -16,6 +16,12 @@
     refreshonly => false,
   }
 
+  pulpcore::admin { 'reset-admin-password --random':
+    unless      => 'python3-django-admin dumpdata | grep "auth.user"',
```

Comment:
This dumps the entire database which can be very expensive. You can dump data just for a single model (https://docs.djangoproject.com/en/3.0/ref/django-admin/#dumpdata) so it think this would work:
```suggestion
    unless      => 'python3-django-admin dumpdata auth.User | grep "auth.user"',
```

Reply:
This works. Updated.. :+1: